src/compiler.rs

```diff
@@ -24,20 +24,31 @@ const NO_CODE_PROVIDED: &str = "No code has been provided to the compiler";
 const AMBER_DEBUG_PARSER: &str = "AMBER_DEBUG_PARSER";
 const AMBER_DEBUG_TIME: &str = "AMBER_DEBUG_TIME";
 
+pub struct CompilerOptions {
+    pub no_proc: Vec<String>,
+    pub minify: bool,
+}
+
+impl Default for CompilerOptions {
+    fn default() -> Self {
+        Self {
+            no_proc: vec![String::from("*")],
```

Comment:
`CompilerOptions` is used everywhere `AmberCompiler` is used, which is to say almost everywhere.  I do not think the `--no-proc` and `--minify` options make sense outside a `cargo check` or `cargo build` scenario, so we disable postprocessors for other subcommands.  If anyone thinks we need to run postprocessors by default with `cargo run`, I can add a command line option for that.

Reply:
i think it would make sense to add an option for `run` for disabling postprocessors, in case there are problems with it

Reply:
My point is that postprocessors are currently enabled only for `check` and `build`, not for `run` subcommands. Your response implies that they should be enabled by default for `run`?

Reply:
I don't use postprocessors at all. Moreover I think that for debugging purposes they should be disabled so that we can see what's actually being returned by amber compiler. Running them make sense when compiling amber in `--release` mode. Wdyt?